charts/epinio/100.0.0+up0.9.1/questions.yml

```diff
@@ -0,0 +1,160 @@
+questions:
+- variable: email
+  label: Email
+  description: "Email to use for getting notifications about your certificates"
+  type: strings
+  required: false
+  group: "General settings"
+- variable: certManagerNamespace
+  label: Cert-manager namespace
+  description: "Namespace where cert-manager is installed in"
+  type: strings
+  required: false
+  group: "Advanced settings"
+- variable: ingress.ingressClassName
+  label: Ingress class name for the server
+  description: "Set a class name to select the ingress controller you want to use for the server"
+  type: strings
+  group: "Advanced settings"
+- variable: server.ingressClassName
+  label: Ingress class name for apps
+  description: "Set a class name to select the ingress controller you want to use for your apps"
+  type: strings
+  group: "Advanced settings"
+- variable: serviceCatalog.enableDevServices
+  label: Enable catalog services for development
+  default: "true"
+  description: "Enables services in the Epinio service catalog, meant to be used in development (because they are running in-cluster)"
```

Comment:
@juadk :bell: I believe that you tested the `questions.yml` when you wrote it.
Do you have answers here ?

Reply:
It's been a long time I didn't test the `questions.yml`. Let me check / remember how I did it and I can check the render of that.

Reply:
And here is the answer:
![image](https://user-images.githubusercontent.com/6025636/171599423-ae01d2a4-7079-430c-a68c-bd9dac823e69.png)

Reply:
I don't know what you think about it, maybe better removing `(because they are running in-cluster)`.

Reply:
Thanks to @juadk for the screenshot

> Looking at how long this description is, I wonder if it will looks off-putting in the questions.yml view...

I believe that is answered now as "looking good".
Because the long list of settings is sorted into 5 groups the user can switch between.